src/main/java/com/oxywire/oxytowns/events/TownDisbandEvent.java

```diff
@@ -0,0 +1,42 @@
+package com.oxywire.oxytowns.events;
+
+import com.oxywire.oxytowns.entities.impl.town.Town;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.HandlerList;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Called when a town has been disbanded by a player.
+ */
+public class TownDisbandEvent extends TownPlayerEvent implements Cancellable {
+    private static final HandlerList HANDLERS = new HandlerList();
+
+    /** The player disbanding the town */
```

Comment:
Do we need this context? Towns will only ever be naturally disbanded by the mayor or a server admin